internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,157 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"testing"
+	"time"
+)
+
+func TestNewAsset(t *testing.T) {
+	type args struct {
+		asset *asset.Asset
+	}
+
+	newAsset, _ := asset.New().
+		NewID().
+		Team(id.NewTeamID()).
+		Name("test").
+		Size(10).
+		URL("test_url").
+		Build()
+
+	tests := []struct {
+		name  string
+		args  args
+		want  *AssetDocument
+		want1 string
+	}{
+		{
+			name: "new asset",
+			args: args{
+				asset: newAsset,
+			},
+			want: &AssetDocument{
+				ID:          newAsset.ID().String(),
+				CreatedAt:   newAsset.CreatedAt(),
+				Team:        newAsset.Team().String(),
+				Name:        newAsset.Name(),
+				Size:        newAsset.Size(),
+				URL:         newAsset.URL(),
+				ContentType: newAsset.ContentType(),
+			},
+			want1: newAsset.ID().String(),
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			got, got1 := NewAsset(tt.args.asset)
+			assert.Equalf(t, tt.want, got, "NewAsset(%v)", tt.args.asset)
+			assert.Equalf(t, tt.want1, got1, "NewAsset(%v)", tt.args.asset)
+		})
+	}
+}
+
+func TestAssetDocument_Model(t *testing.T) {
+	type fields struct {
+		ID          string
+		CreatedAt   time.Time
+		Team        string
+		Name        string
+		Size        int64
+		URL         string
+		ContentType string
+	}
+
+	newFields := fields{
+		ID:          "id",
+		CreatedAt:   time.Time{},
+		Team:        "team",
+		Name:        "name",
+		Size:        10,
+		URL:         "test",
+		ContentType: "content type",
+	}
+
+	aid, _ := id.AssetIDFrom(newFields.ID)
+	tid, _ := id.TeamIDFrom(newFields.Team)
+
+	newAsset, _ := asset.New().
+		ID(aid).
+		CreatedAt(newFields.CreatedAt).
+		Team(tid).
+		Name(newFields.Name).
+		Size(newFields.Size).
+		URL(newFields.URL).
+		ContentType(newFields.ContentType).
+		Build()
+
+	tests := []struct {
+		name    string
+		fields  fields
```

Comment:
I prefer this style since It's more redundant than necessary.

```suggestion
		target *AssetDocument
```